sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/_models.py

```diff
@@ -168,92 +180,71 @@ def __init__(self, feature_id, enabled, filters=[], **kwargs):  # pylint: disabl
         self.etag = kwargs.get("etag", None)
         self.description = kwargs.get("description", None)
         self.display_name = kwargs.get("display_name", None)
-        self.value = kwargs.get("value", {"enabled": enabled, "conditions": {"client_filters": filters}})
-
-    def _validate(self):
-        # type: () -> None
-        if not self.key.startswith(self.key_prefix):
-            raise ValueError("All FeatureFlagConfigurationSettings should be prefixed with {}.".format(self.key_prefix))
-        if not (self.value is None or isinstance(self.value, dict)):
-            raise ValueError("Expect 'value' to be a dictionary.")
-
-    @property
-    def enabled(self):
-        # type: () -> Union[None, bool]
-        self._validate()
-        if self.value is None or "enabled" not in self.value:
-            return None
-        return self.value["enabled"]
-
-    @enabled.setter
-    def enabled(self, new_value):
-        # type: (bool) -> None
-        self._validate()
-        if self.value is None:
-            self.value = {}
-        self.value["enabled"] = new_value
+        self.filters = kwargs.get("filters", [])
+        self.enabled = kwargs.get("enabled", None)
+        self._value = json.dumps({"enabled": self.enabled, "conditions": {"client_filters": self.filters}})
 
     @property
-    def filters(self):
-        # type: () -> Union[None, List[Any]]
-        self._validate()
-        if self.value is None:
-            return None
-        try:
-            return self.value["conditions"]["client_filters"]
-        except KeyError:
-            pass
-        return None
-
-    @filters.setter
-    def filters(self, new_filters):
-        # type: (List[Dict[str, Any]]) -> None
-        self._validate()
-        if self.value is None:
-            self.value = {}
+    def value(self):
+        if self.enabled is None and self.filters is None:
+            return self._value
```

Comment:
Why we cache the value instead of re-generating it every time?

How can we know if enabled or filters changed or not?

Reply:
Updated this. We have to store value in case the value is set to malformed json

Reply:
if value is malformed, is it still a FeaFlagConfigSetting?

What's the behavior of filters.setter if it is malformed?

Reply:
If the user is setting `value`, they lose all protections, we assume they know what they are doing. If the value is malformed and then a user sets filters, the filters will be lost. The only way to get a malformed json is by setting value directly and in that case we assume the customer understands what they are doing.

Reply:
Then we need another variable to store if it is malformed.

Or how can we track filters is modified or not?